tests/purs/failing/2616.out

```diff
@@ -16,6 +16,7 @@ while applying a function [33mcompare[0m
   of type [33mOrd t0 => t0 -> t0 -> Ordering[0m
   to argument [33m$l2[0m
 while inferring the type of [33mcompare $l2[0m
+while matching label [33mcompare[0m
```

Comment:
This is also a bad hint. There is no `compare` label in the code; the identifier `compare` doesn't even appear.